holidayextras/ui-toolkit
# Convert the steppers css to use flexbox instead

#### What does this PR do? (please provide any background)

Converts the current stepper styles to use Jodies flexbox styles instead.

#### What tests does this PR have?

None

#### How can this be tested?

run the docs and check it x-browser to make sure it still appears the same as it currently does.

#### Screenshots / Screencast

http://grab.by/LMlm

#### What gif best describes how you feel about this work?

![](https://media.giphy.com/media/sSsKWDd1hcOc0/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
